policy/composer.go

```diff
@@ -192,6 +168,187 @@ func (opt *ruleComposerImpl) registerVariable(ctx *cel.OptimizerContext, v *Comp
 	opt.nextVarIndex++
 }
 
+// sortedVariables returns the variables ordered by their declaration index.
 func (opt *ruleComposerImpl) sortedVariables() []varIndex {
 	return opt.varIndices
 }
+
+// outputStep interface represents a policy output expression.
+type outputStep interface {
```

Comment:
nit: why "step" if it represents an expression? (maybe expanding the comment is enough to explain)